playground/TestShop/TestShop.AppHost/aspire-manifest.json

```diff
@@ -203,9 +203,23 @@
     },
     "apigateway": {
       "type": "container.v0",
-      "image": "mcr.microsoft.com/dotnet/nightly/yarp:latest",
+      "image": "mcr.microsoft.com/dotnet/nightly/yarp:2.3.0-preview.3",
+      "entrypoint": "dotnet",
+      "args": [
+        "/app/yarp.dll"
+      ],
       "env": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "services__catalogservice__http__0": "{catalogservice.bindings.http.url}",
+        "REVERSEPROXY__ROUTES__route0__MATCH__PATH": "/catalog/{**catch-all}",
+        "REVERSEPROXY__ROUTES__route0__CLUSTERID": "cluster_catalogservice",
+        "REVERSEPROXY__ROUTES__route0__TRANSFORMS__0__PathRemovePrefix": "/catalog",
+        "REVERSEPROXY__ROUTES__route1__MATCH__PATH": "/basket/{**catch-all}",
+        "REVERSEPROXY__ROUTES__route1__CLUSTERID": "cluster_basketservice",
+        "REVERSEPROXY__ROUTES__route1__TRANSFORMS__0__PathRemovePrefix": "/basket",
+        "REVERSEPROXY__CLUSTERS__cluster_catalogservice__DESTINATIONS__destination1__ADDRESS": "http://_http.catalogservice",
```

Comment:
@benjaminpetit is this right? Shouldn’t it be {scheme}://_{endpointname}.{resource}